### PR TITLE
updated trends-chart with updated response data

### PR DESCRIPTION
**Pull request by gnehapk**

### CODE REVIEW

**Comment by gnehapk** [comment]
@kamleshverma1 @cloudbehl Please review

**Comment by kamleshverma1** [comment]
look good to me

File: src/commons/services/utils.js [hunk 1]
```diff
@@ -311,6 +310,25 @@
             }
         };
 
+        vm.getTrendData = function() {
```
Comment (cloudbehl) [inline src/commons/services/utils.js]: why this extra method for get request when the method is already there? If it is for the testing purpose, then please don't include it in code. As this code needs to remove after the API is there.

Comment (gnehapk) [inline src/commons/services/utils.js]: @cloudbehl About which method are you talking?

**Comment by gnehapk** [comment]
@kamleshverma1 @cloudbehl I have updated the PR. Please review

**Pull request merged**